domain/infosync/info.go

```diff
@@ -804,3 +805,33 @@ func getServerInfo(id string, serverIDGetter func() uint64) *ServerInfo {
 
 	return info
 }
+
+// PutLabelRule synchronizes the label rule to PD.
+func PutLabelRule(ctx context.Context, rule *label.Rule) error {
+	if rule == nil {
+		return nil
+	}
+
+	is, err := getGlobalInfoSyncer()
+	if err != nil {
+		return err
+	}
+
+	if is.etcdCli == nil {
+		return nil
```

Comment:
Maybe we should add some logs here.

Reply:
I think we can add it in another PR if necessary.

Reply:
It is for testing, where etcd may not be available.